01_fib/fibonacci.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(num int) {
+	num1 := 0
+	num2 := num1 + 1
```

Comment:
Here `num1` is guaranteed to be `0`. You can directly assign 1 to num2. 
Also, as @anzdaddy suggested on other PRs you can combine these two statements to `num1, num2 := 0, 1`